js/interface.js

```diff
@@ -1847,10 +1854,13 @@ var DynamicLists = (function() {
         ? '(Value)'
         : data.value;
 
+      data.valueField = 'Value';
```

Comment:
You shouldn't need to save this into the LFD configuration just because the interface needs this in the template. We don't want to save data that can be derived from other configurations during runtime.

Reply:
@tonytlwu fixed and made changes

Reply:
@AndrRyaz Go ahead. You don't need to wait for my approval when I already gave my perspective 4 days ago on what needs to be done. cc @sofiiakvasnevska